assets/style.css

```diff
@@ -0,0 +1,505 @@
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #f8f5f5;
+  min-height: 100vh;
+}
+
+.navbar {
+  display: none;
+}
+
+.hamburger {
+  width: 25px;
+  height: 25px;
+  margin-bottom: 10%;
+}
+
+/* intro section */
+.intro {
+  background-image: url('./images/header.jpg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.blur {
+  display: flex;
+  flex-direction: column;
+  gap: 1%;
+  padding-left: 2%;
+  padding-right: 3%;
+  padding-top: 3%;
+  padding-bottom: 10%;
+  backdrop-filter: blur(5px);
+  background-color: rgba(255, 255, 255, 0.3);
+}
+
+.welcome {
+  font-size: large;
+  font-weight: bold;
+  letter-spacing: 1px;
+  font-style: italic;
+  color: #5a3f3b;
+}
+
+.title {
+  font-size: xx-large;
+  font-weight: bold;
+  color: #8f6b6b;
+}
+
+.introDescription {
+  border: 4px solid rgb(10 10 10 / 47%);
+  padding: 5%;
+  color: #9f9c9c;
+  margin-top: 10%;
+  margin-bottom: 10%;
+  background-color: rgba(237, 230, 230, 0.9);
+}
+
+.introFooter {
+  font-size: x-large;
+  color: #f8f5f5;
+  padding-left: 3%;
+}
+
+/* Main programs section */
+.programs {
+  background-image: url('./images/dot.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-color: #303440;
+  padding-left: 3%;
+  padding-right: 3%;
+  padding-top: 5%;
+  padding-bottom: 5%;
+}
```

Comment:
- [ ] You did a good job styling your project :+1:, but as a project requirement your page should look almost identical to the original design, outside the content differences. The pages should use the exact same colors, for text, backgrounds, borders, etc, from the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). I've noticed that your page doesn't have any of the colors required, so kindly change the colors from some texts, backgrounds, and borders to match with the Template :+1: 

**Required colors**

![Screenshot from 2023-02-17 08-34-35](https://user-images.githubusercontent.com/105117832/219666548-554244e3-7b4e-492c-a701-0f5b1d177e98.png)